src/main/java/com/andersen/webroomba/model/implementation/RoboticHoover.java

```diff
@@ -85,13 +87,22 @@ public void setInstructions(String instructions) {
 
     @Override
     public void setGrid(RoomGrid grid) {
-             this.grid = grid;
+        this.grid = grid;
     }
 
     @Override
     public void setLidar(Lidar lidar) {
-          this.lidar =lidar;
+        this.lidar = lidar;
     }
 
+    @Override
+    public DirtContainer ejectDirtContainer() {
+        return dirtContainer;
```

Comment:
I think this method should set current dirtContainer to null, at least this is the meaning of this method in my head - when I eject a container its not there anymore until I put it back again :)

Reply:
Knowing this - more tests should follow :)